PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/sparkserverless/SparkServerlessClusterOpsCtrl.java

```diff
@@ -44,42 +44,42 @@ public SparkServerlessClusterOpsCtrl(@NotNull SparkServerlessClusterOps sparkSer
         this.sparkServerlessClusterOps.getDestroyAction()
                 .observeOn(ideSchedulers.dispatchUIThread())
                 .subscribe(triplet -> {
-                    LOG.info(String.format("Destroy message received. AdlAccount: %s, cluster: %s, currentNode: %s",
+                    log().info(String.format("Destroy message received. AdlAccount: %s, cluster: %s, currentNode: %s",
                             triplet.getLeft().getName(), triplet.getMiddle(), triplet.getRight().getName()));
                     SparkServerlessClusterDestoryDialog destroyDialog = new SparkServerlessClusterDestoryDialog(
                             triplet.getRight(), triplet.getMiddle());
                     destroyDialog.show();
-                }, ex -> LOG.error(ex.getMessage(), ex));
+                }, ex -> log().warn(ex.getMessage(), ex));
 
         this.sparkServerlessClusterOps.getProvisionAction()
                 .observeOn(ideSchedulers.dispatchUIThread())
                 .subscribe(pair -> {
-                    LOG.info(String.format("Provision message received. AdlAccount: %s, node: %s",
+                    log().info(String.format("Provision message received. AdlAccount: %s, node: %s",
                             pair.getLeft().getName(), pair.getRight().getName()));
                     SparkServerlessProvisionDialog provisionDialog = new SparkServerlessProvisionDialog(
                             pair.getRight(), pair.getLeft());
                     provisionDialog.show();
-                }, ex -> LOG.error(ex.getMessage(), ex));
+                }, ex -> log().warn(ex.getMessage(), ex));
 
         this.sparkServerlessClusterOps.getMonitorAction()
                 .observeOn(ideSchedulers.dispatchUIThread())
                 .subscribe(pair -> {
-                    LOG.info(String.format("Monitor message received. cluster: %s, node: %s",
+                    log().info(String.format("Monitor message received. cluster: %s, node: %s",
                             pair.getLeft(), pair.getRight()));
```

Comment:
>pair.getLeft(), pair.getRight())); [](start = 28, length = 34)

Either implement a toString method for AzureSparkServerlessCluster, SparkServerlessClusterNode? Or add getName() here?